src/intelligence_layer/core/model.py

```diff
@@ -601,17 +636,50 @@ def to_instruct_prompt(
         instruction: str,
         input: Optional[str] = None,
         response_prefix: Optional[str] = None,
+        instruction_controls: Optional[Sequence[TextControl]] = None,
+        input_controls: Optional[Sequence[TextControl]] = None,
     ) -> RichPrompt:
-        return self.to_chat_prompt(
+        """Method to use a Chat Model like an Instruct Model`.
+
+        Allows the implementation of a custom prompt format for the specific model in use.
```

Comment:
I know you copied this and this is how it was before, but this sentence doesn't tell me anything really

Reply:
Changed